### ISSUE TITLE
Multi-sig transaction - update commission rates doesn't work

### ISSUE THREAD

**Issue by mh-concordium** [Concordium/concordium-desktop-wallet#332]
**Bug Description**
Attempt to update commission rates with multi-sig transaction will fail, it seems that such a change is not recognized as a change to the pool. I assume that priority for this is low since  commissions are fixed on Mainnet.

**Steps to Reproduce**
Add baker with multi-sig ->try to change commission rates.
Update baker -> change commission rates again.

**Expected Result**
Values should be changed accordingly as selected by user.

**Actual Result**
Values are not changed.

**Versions**
- Software Version: 1.5.0
- OS: Windows

https://user-images.githubusercontent.com/98821384/177545128-a38ccb5b-f05d-45c3-8b62-38f8edb44343.mp4

**Comment by shjortConcordium** [comment]
Yeah, it should either skip the commission page, or take the changes into account.
